tests/help.rn

```diff
@@ -0,0 +1,33 @@
+class Test
+{
+    property1 = "Hello, World!"
+
+    fun __constructor__()
+    {
+
+    }
+
+    fun func1(arg1,arg2)
```

Comment:
What if there is some default value `fun func1(arg1=3, arg2=null)` ?

Reply:
Currently it will just write the names of the args, but not the default values

Reply:
> Currently it will just write the names of the args, but not the default values

Well, if you don't want to implement something now, then create an issue to remember it for later patch.

Reply:
I think you missed to set it for class methods.